ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/boolean.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'appBooleanHumanView'
+})
+
+export class AppBooleanHumanViewPipe implements PipeTransform {
+  transform(value: any, args?: any): any {
+
+    if (value !== undefined && value !== null) {
+      return value ? 'Yes' : 'No';
+    }
+
+    return '';
```

Comment:
So if value is null or undefined we get an empty string? Should we raise this as an error or something, might be tough to troubleshoot otherwise?

Reply:
done